scripts/eqtl_hail_batch/generate_eqtl_spearman.py

```diff
@@ -28,6 +28,7 @@
 TOB_WGS = dataset_path('mt/v7.mt/')
 FREQ_TABLE = dataset_path('joint-calling/v7/variant_qc/frequencies.ht/', 'analysis')
 VEP_ANNOTATION = dataset_path('tob_wgs_vep/v1/vep105_GRCh38.mt/')
```

Comment:
This should point to `vep104`, right?

Reply:
Yes, thanks for that. I've fixed the path